airflow/config_templates/default_airflow.cfg

```diff
@@ -840,3 +840,7 @@ fs_group =
 # The worker pods will be given these static labels, as well as some additional dynamic labels
 # to identify the task.
 # Should be supplied in the format: key = value
+
+[kubernetes_worker_resources]
+# EWT-290: This is added adhoc basis to configure the Airflow worker resources.
+# This should be removed when the similar functionality is available with Airflow upgrade.
```

Comment:
we can add "airflow release > 1.10.10" maybe for future ref

Reply:
I have added some more information in the JIRA. Can you please check that.